docs/source/asr/configs.rst

```diff
@@ -1120,3 +1120,160 @@ Depending on the type of model, there may be extra steps that must be performed
 
 * CTC Models - `Examples directory for CTC Models <https://github.com/NVIDIA/NeMo/blob/stable/examples/asr/asr_ctc/README.md>`_
 * RNN Transducer Models - `Examples directory for Transducer Models <https://github.com/NVIDIA/NeMo/blob/stable/examples/asr/asr_transducer/README.md>`_
+
+Multi-task AED Models
+------------------
+
+The config for multitask AED models (e.g., Canary models that can perform both ASR and translation tasks) is at ``<NeMo_git_root>/examples/asr/conf/speech_multitask/fast-conformer_aed.yaml``. Multi-task AED models are built with an encoder-decoder architecture utilizing FastConformer for the encoder and Transformer for the decoder.
+
+Various sections of the config of `Multi-task AED Models <./models.html#_AED_model>`__ are as follows:
+
+* Model initialization (``init_from_nemo_model``)
+* Dataset configs (``train_ds``, ``validation_ds``, and ``test_ds``)
+* Special tokenizer config (``spl_tokens``)
+* Tokenizer config (``tokenizer``)
+* Prompt format (``prompt_format``)
+* Model defaults (``model_defaults``)
+* Audio preprocessor (``preprocessor``)
+* Augmentation (``spec_augment``)
+* FastConformer encoder (``encoder``)
+* Optional intermediate transformer encoder (``transf_encoder``)
+* Transformer decoder (``transf_decoder``)
+* Label prediction head (``head``)
+* Decoding strategy (``decoding``)
+* Loss config (``loss``)
+* Optimizer (``optim``)
+* Training config (``trainer``)
+* Experiment manager (``exp_manager``)
+
+While most of the sections are similar to other ASR models, the multi-task AED model has a few unique sections:
+
+Model Initialization
+~~~~~~~~~~~~~~~~~~~~
+
+For larger models (1B+ params), initialization from a pretrained encoder is recommended for better stability and convergence:
+
+.. code-block:: yaml
+
+    init_from_nemo_model:
+      model0:
+        path: "<path to pretrained model>"
+        include: ["encoder"]
+        exclude: ["encoder.pre_encode.out"]
+
+The ``include`` parameter specifies which components to load from the pretrained model. In this case, only the encoder weights are loaded. The ``exclude`` parameter allows you to skip specific sub-components during loading - here, the pre-encoder output layer is excluded to allow for architectural modifications.
+
+
+Dataset Configurations
+~~~~~~~~~~~~~~~~~~~~
+
+The multi-task AED models support only Lhotse-based data loading. Datasets can be configured using either manifest files or Lhotse YAML configs:
+
+.. code-block:: yaml
+
+    train_ds:
+      use_lhotse: true
+      input_cfg: "<path to lhotse config>"
+      batch_duration: 600
+      max_duration: 40
+      use_bucketing: True
+      num_buckets: 30
+      bucket_duration_bins: [3.79, 4.82, 5.688, ..., 35.827]
```

Comment:
Provide info / link to calcualte bucket duration bins (ir just update below sentence to mention that to get the list of floats, follow this link)